x/auth/signing/verify_test.go

```diff
@@ -0,0 +1,69 @@
+package signing_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	abci "github.com/tendermint/tendermint/abci/types"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/simapp"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/auth/ante"
+	"github.com/cosmos/cosmos-sdk/x/auth/signing"
+	"github.com/cosmos/cosmos-sdk/x/auth/types"
+)
+
+func TestVerifySignature(t *testing.T) {
+	priv, pubKey, addr := types.KeyTestPubAddr()
+
+	const (
+		from = "test_sign"
+		backend = "test"
```

Comment:
What is backend? Is this used?

Reply:
Oh, missed to clean this up. It was used for keyring before, will remove it